TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -97,6 +97,11 @@ public void LoadCpyCopyNameMap(string cpyCopyNameMapFilePath)
         }
 #endif
 
+        /// <summary>
+        /// Is Sql Code Analysis allowed.
+        /// </summary>
+        public bool UseSqlCodeAnalysis = false;
```

Comment:
Not needed as this would be implemented in our private analyzer.

Reply:
Ok it will be a task, this is for the POC

Reply:
Why would it be a task? Even for the POC we don't need it.

Reply:
done in a98dcf1